react/ShelfItem.js

```diff
@@ -6,28 +6,29 @@ import PropTypes from 'prop-types'
  */
 class ShelfItem extends Component {
   render() {
-    const { productName, items, imageHeight } = this.props
-    const imagePath = items[0].images[0].imageUrl
-    const price = items[0].sellers[0].commertialOffer.Price
+    const { name, imageUrl, listPrice } = this.props
+
     return (
-      <div className="items-center flex flex-column pointer">
-        <h4 className="w-90 tc truncate mid-gray" title={productName}>
-          {productName}
-        </h4>
-        <img height={imageHeight} src={imagePath.replace('http:', '')} />
-        <h5 className="near-black">R$ {price.toFixed(2)}</h5>
+      <div className="ph4 grow ">
```

Comment:
Isn't here that product-summary will be injected?

Reply:
No, this component is never used, i need to remove it :x

Reply:
So, remove it. 😄